docs/releasenotes/index.rst

```diff
@@ -16,6 +16,7 @@ Next Release
 * The new :ref:`CloudFormation installer <cloudformation>` has been made available, to provide a far simpler installation experience for users on AWS.
 * The :ref:`Flocker plugin for Docker <plugin>` should support the direct volume listing and inspection functionality added to Docker 1.10.
 * Fixed a regression that caused block device agents to poll backend APIs like EBS too frequently in some circumstances.
+* Datasets can now be moved off of shutdown EC2 instances. Previously they could only be moved off of terminated instances.
```

Comment:
I've tried it on Rackspace manually. It doesn't work due to Rackspace bugs. It *does* work on OpenStack elsewhere (at least on Swisscom's cluster) but I don't want to claim that publicly until we can enable the acceptance tests.